sdks/java/core/src/main/java/org/apache/beam/sdk/io/LocalFileSystem.java

```diff
@@ -17,8 +17,8 @@
  */
 package org.apache.beam.sdk.io;
 
+import static com.google.common.io.Files.fileTraverser;
```

Comment:
You want to stick with the org.apache.beam.vendor.guava.v20_0.com.google.common.io.Files.fileTraverser import path since we are specifically vendoring our guava dependency so it doesn't conflict with user supplied versions of guava.

Reply:
I see. I'm a Google engineer and I want to get rid of this dependency so that I can delete fileTreeTraverser(). Could I depend on guava.v26_0 here instead? I assume that would need some changes in some dependency configuration file?

Reply:
No, since an Apache Beam vendored version of Guava v26_0 doesn't exist on Maven central.

We first need to get the vote to pass (you are allowed to take a look at the proposed release process being suggested on that thread and provide a non binding vote which would help the process along). This would allow for preparing a release candidate containing a vendored version of Guava 26 for Apache Beam. After that is out, we would need to update and merge #8899.

Reply:
Where is it that I can cast a non-binding vote?

Reply:
You will need to subscribe to the mailing list by sending a blank email to dev-subscribe@beam.apache.org

Goto https://lists.apache.org/thread.html/603296cdaf673d38eb3fd1d7d414d8f4f3e4658a190ff84020101ba2@%3Cdev.beam.apache.org%3E and click on reply.

Reply:
Done and thanks!

Reply:
The vote for the release process has passed. I have created a release candidate for vendored Guava 26.0 and started a vote for the release. Feel free to validate and vote on the release: https://lists.apache.org/thread.html/a821404a9cbbbefab872fb339774547c9183136f452d80dc41c37abe@%3Cdev.beam.apache.org%3E

After this, all that remains is updating #8899 resolving any conflicts that have happened since the PR was created.

Reply:
It might make sense to close this PR in favor of #8899 since it encompasses this change and much more.

Reply:
Jep, #8899 solves my issue. Thanks!

Reply:
The vendored artifacts have been released and #8899 has been updated accordingly.

Reply:
Many thanks!